fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -304,17 +304,23 @@ public void executeQuery(String originStmt) throws Exception {
             try {
```

Comment:
We should write unit test to test case.
I suggest you extract this part of method, and write unit test to make sure this logic works